modules/shared/components/molecules/Privacy/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import classNames from 'classnames';
+import Toggler from '../../atoms/toggler';
+import type { FC, ReactElement } from 'react';
+import type { PrivacyTypes } from './PrivacyTypes';
+import styles from './Privacy.module.css';
+
+/** 
+* @example : <Privacy privacyOptions={["friends","family"]} withDivider togglerIsChecked={togglerIsChecked} handleTogglerChange={handleTogglerChange} handleSelectChange={handleSelectChange} />
+* @Parameters privacyOptions, withDivider, togglerIsChecked, handleTogglerChange and handleSelectChange are required for Toggler component to function, The prop togglerLabel and privacySelectorLabel is optional.
+* @Notice you will use the handleChange functions to get and use the value of the toggler and the select
+* @ThingsNeededToBeDoneInParentComponent
+const [togglerIsChecked,setTogglerIsChecked] = useState(false);
+cosnt handleTogglerChange = (e: React.ChangeEvent<HTMLInputElement>):void => { setTogglerIsChecked(e.target.checked) }<Test>;
+const handleSelectChange = (e: React.ChangeEvent<HTMLSelectElement>):void => {console.log(e.target.value)};
+*/
+const Privacy: FC<PrivacyTypes.Props> = ({
+  togglerLabel = 'Hide my identity',
+  privacySelectorLabel = 'Privacy:',
+  privacyOptions,
+  withDivider,
+  togglerIsChecked,
+  handleTogglerChange,
+  handleSelectChange,
+}): ReactElement => {
+  const privacySelectorContainer = classNames(styles.privacySelectorContainer, {
+    [styles.privacySelectorContainerWithoutDivider]: !withDivider,
+  });
+  return (
+    <div className={styles.privacyContainer}>
+      <div className={styles.hideMyIdentityContainer}>
+        <Toggler
+          size="sm"
+          id="hideMyIdentityToggler"
+          checked={togglerIsChecked}
+          onChange={handleTogglerChange}
+        />
+        <span className={styles.hideMyIdentityText}>{togglerLabel}</span>
+      </div>
+      {withDivider && (
+        <div data-testid="dividerTestId" className={styles.divider} />
+      )}
+      <div className={privacySelectorContainer}>
+        <span className={styles.privacySelectorText}>
+          {privacySelectorLabel}
+        </span>
+        <select
+          data-testid="selectTestId"
+          onChange={handleSelectChange}
+          className={styles.privacySelector}
+        >
+          {privacyOptions.map((option) => {
+            return (
+              <option
+                key={option}
+                data-testid={option}
+                className={styles.privacySelectorOptions}
+                value={option}
+              >
+                {option}
+              </option>
+            );
+          })}
+        </select>
```

Comment:
According to Figma, Please consider using the Button(Text + Right Icon) component instead of repeating the functionality and the styles especially at the responsiveness part in the future.
![Screenshot from 2021-05-20 16-22-57](https://user-images.githubusercontent.com/52051312/118995926-d29a8e80-b987-11eb-9767-cc9c762dd8c3.png)

![Screenshot from 2021-05-20 16-15-21](https://user-images.githubusercontent.com/52051312/118994509-b0544100-b986-11eb-9932-ae66d0b67393.png)